arch/arm/soc/atmel_sam/common/soc_gpio.c

```diff
@@ -44,10 +47,22 @@ static void configure_input_attr(Pio *pio, u32_t mask, u32_t flags)
 	if ((flags & SOC_GPIO_IN_FILTER_MASK) != 0) {
 		if ((flags & SOC_GPIO_IN_FILTER_MASK) == SOC_GPIO_IN_FILTER_DEBOUNCE) {
 			/* Enable de-bounce, disable de-glitch */
+#if defined PIO_IFSCER_P0
 			pio->PIO_IFSCER = mask;
+#elif defined PIO_DIFSR_P0
+			pio->PIO_DIFSR = mask;
+#else
+#error "Unsupported Atmel SAM MCU series"
```

Comment:
I'll add the following code at the beginning of the file and remove remaining #error messages.
```
#if    (!defined(PIO_IFSCER_P0) && !defined(PIO_DIFSR_P0)) \
    || (!defined(PIO_IFSCDR_P0) && !defined(PIO_SCIFSR_P0)) \ 
    || (!defined(PIO_ABCDSR_P0) && !defined(PIO_ABSR_P0)) 
#error "Unsupported Atmel SAM MCU series"
#endif
```